composer.json

```diff
@@ -1,13 +1,16 @@
 {
     "type": "project",
     "license": "MIT",
-    "minimum-stability": "dev",
+    "minimum-stability": "stable",
     "prefer-stable": true,
     "require": {
-        "hwi/oauth-bundle": "^0.6",
-        "msgphp/eav-bundle": "^1.0",
-        "msgphp/user-bundle": "^1.0",
-        "php-http/guzzle6-adapter": "^1.2",
+        "hwi/oauth-bundle": "dev-master@dev",
+        "msgphp/domain": "^1.0@dev",
+        "msgphp/eav": "^1.0@dev",
```

Comment:
`"@dev"` (version constraint is defined by the bundle)

Reply:
TIL